proposals/lip-0052.md

```diff
@@ -1296,6 +1272,54 @@ def createEscrowEntry(
         key = receivingChainID + nftID
 ```
 
+#### verifyTransferInternal
+
+```python
+def verifyTransferInternal(
+    senderAddress: Address,
+    recipientAddress: Address,
+    nftID: NFTID,
+    emitEvent: bool
+) -> None:
+
+    nft = getNFT(nftID)
```

Comment:
I think we should handle the possible exception from this function by emitting an event with `RESULT_NFT_DOES_NOT_EXIST` if the call to `getNFT` fails.

Reply:
Done in [this commit](https://github.com/LiskHQ/lips/pull/477/commits/876ae2c110612053e85a0c54fb65574a7e1b4bc1).